src/main/java/uk/gov/companieshouse/web/companydocuments/controller/CompanyDetailsController.java

```diff
@@ -27,15 +27,15 @@ public CompanyDetailsController(CompanyService companyService,
     public String getCompanyDetails(@PathVariable String companyNumber,
                                     Model model) {
 
+        model.addAttribute("backButton", NavigationUrls.COMPANY_LOOKUP_FORWARD_TO_COMPANY_DETAILS);
```

Comment:
How about we make the `backButton` string literal a `protected static final String` constant in the `BaseController` since it's used in multiple places?

Reply:
I'm happy to do that instead 👍